### ISSUE TITLE
Comments: Deploy Updates

### ISSUE THREAD

**Issue by jayair** [AnomalyInnovations/serverless-stack-com#68]

**Comment by deepseafishing** [comment]
@jayair 
hi, jayair 
I was wondering what "CloudFront caches our objects in it’s edge locations" meant?

so when I deploy again with the changes in buttons, I can see the changes in s3-website url but can't see them in my own linked up domain name. ( the one we just set up) 

Can you explain a little bit more on this?

Thank you so much. I really appreciate your tutorial.

**Comment by jayair** [comment]
@deepseafishing Yeah CloudFront is a CDN so our assets are cached on edge servers that Amazon has. That's why we need to invalidate the CloudFront cache. It can take a minute or two to invalidate it but you should see your changes soon.

**Comment by lepirlouit** [comment]
Why is it required to clear the cache of the redirecting cloudfront ?
The redirected page url in S3 didn't change. Isn't it ?

**Comment by jayair** [comment]
@lepirlouit Yeah you don't have to. We just do it for completeness sake.

**Comment by jayair** [comment]
This issue was moved to https://discourse.serverless-stack.com/t/comments-deploy-updates/16
